buildpipeline/pipeline.json

```diff
@@ -38,7 +38,8 @@
             "PB_TargetArchitecture": "arm", 
             "PB_AdditionalBuildArguments":"-TargetArchitecture=arm -DistroRid=linux-arm -DisableCrossgen=true -PortableBuild=true -SkipTests=true -CrossBuild=true",
             "PB_CrossBuildArgs": "-e ROOTFS_DIR ",
-            "PB_PortableBuild": "true" 
+            "PB_PortableBuild": "true",
+            "PublishDebToolToFeed": "true"
```

Comment:
We are publishing this during the `linux-arm` build?  That seems like an odd place.